week-2-practise/level-1/3-sort.js

```diff
@@ -2,7 +2,7 @@
 // TIP: use an array method
 
 var letters = ["a", "n", "c", "e", "z", "f"];
-var sortedLetters; // ONLY EDIT THIS LINE
+var sortedLetters = ["a", "c", "e", "f", "n", "z"]; // ONLY EDIT THIS LINE
```

Comment:
This manually sort the letters. The intent is to sort the var letters using the sort function.